lib/gollum-lib/macro/navigation.rb

```diff
@@ -6,7 +6,7 @@ def render(title = "Navigate in the TOC", toc_root_path = ::File.dirname(@page.p
         if @wiki.pages.size > 0
           list_items = @wiki.pages.map do |page|
             if page.url_path.start_with?(toc_root_path)
-              path_display = full_path ? page.url_path_display  : page.url_path_display.sub(toc_root_path,"").sub(/^\//,'')
+              path_display = full_path ? page.url_path_display  : page.url_path_display.sub(toc_root_path.gsub("-", " "), "").sub(/^\//,'')
```

Comment:
The problem is the `toc_root_path`, which is dashed as `Research_Topic_1` in this case. So I undash it to make the stripping work as expected. Note that for single-word `toc_root_path` such as "ResearchTopic1", there is no problem.